src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* Apa saja editor yang paling populer
+* Cara memformat kode Anda secara otomatis
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## Editor Anda {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) adalah salah satu editor yang paling populer yang digunakan saat ini. Memiliki pasar ekstensi yang besar dan terintegrasi dengan baik dengan layanan populer seperti GitHub. Sebagian besar fitur yang tercantum di bawah ini juga dapat ditambahkan ke VS Code sebagai ekstensi, sehingga membuatnya sangat dapat dikonfigurasi!
 
-Other popular text editors used in the React community include:
+Editor teks populer lainnya yang digunakan dalam komunitas React meliputi:
 
-* [WebStorm](https://www.jetbrains.com/webstorm/) is an integrated development environment designed specifically for JavaScript.
-* [Sublime Text](https://www.sublimetext.com/) has support for JSX and TypeScript, [syntax highlighting](https://stackoverflow.com/a/70960574/458193) and autocomplete built in.
-* [Vim](https://www.vim.org/) is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X.
+* [WebStorm](https://www.jetbrains.com/webstorm/) adalah lingkungan pengembangan terpadu yang dirancang khusus untuk JavaScript.
+* [Sublime Text](https://www.sublimetext.com/) memiliki dukungan untuk JSX dan TypeScript, [*syntax highlighting*](https://stackoverflow.com/a/70960574/458193) dan penyelesaian otomatis yang sudah *built-in*.
+* [Vim](https://www.vim.org/) adalah editor teks yang sangat dapat dikonfigurasi yang dirancang untuk membuat membuat dan mengubah jenis teks apa pun menjadi sangat efisien. Ini disertakan sebagai "vi" dengan sebagian besar sistem UNIX dan dengan Apple OS X.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## Fitur editor teks yang direkomendasikan {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+Beberapa editor dilengkapi dengan fitur-fitur ini secara *built-in*, tetapi yang lain mungkin memerlukan penambahan ekstensi. Periksa untuk melihat dukungan yang disediakan editor pilihan Anda untuk memastikan!
 
-### Linting {/*linting*/}
+### Pemeriksaan {/*linting*/}
```

Comment:
Karena *Linting* ini kan agak2 refer ke istilah teknis, bagaimana kalau kita tetap mempertahankan kata-kata *Linting* nya juga?

Reply:
Gua sebagai orang teknis pun bisa kehilangan maksud ketika "Linting" langsung ditranslasi pake kata "Pemeriksaan".

Reply:
yeah, that's what I thought. I can't find the word in the [glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium)

Reply:
done, thanks for the suggestion!

Changes:
Pemeriksaan -> *Linting*
Pemeriksa -> *Linter*